stdlib/public/common/LeafMirrorConformance.gyb

```diff
@@ -0,0 +1,22 @@
+%{
```

Comment:
Can we sink this file into `stdlib/public/core/Mirrors.swift.gyb` so we don't have to do that `parseTemplate`/`executeTemplate` dance?